### ISSUE TITLE
google places javadoc not found 404 & how to use API-KEY undocumented

### ISSUE THREAD

**Issue by mh-trimble** [googleapis/google-cloud-java#10904]
the javadocs linked for this library are not found:

https://cloud.google.com/java/docs/reference/google-maps-places/latest/overview

also https://developers.google.com/maps/documentation/places/web-service/get-api-key says that you can use an api key, but don't understand how to use an API key in the context of the library

**Comment by lqiu96** [comment]
Hi @mh-trimble, thanks for reporting this. I asked around internally and I believe the reference documentation for maps libraries are in the process of being migrated to the new URL (something like: `developers.google.com/maps/reference/documentation/{places|routing|address}/...`) and should be up in the next few weeks. Apologies that the transition hasn't been smooth. Can you point us to the place where you found `https://cloud.google.com/java/docs/reference/google-maps-places/latest/overview`?

Also, regarding API-Keys: I believe our client libraries don't have the best built-in support for Api-Keys (as of now). I think we have a way to enable it and are looking into ways to make this better.

CC: @blakeli0 @alicejli if you have a way enable ApiKeys now for client libraries and any updates regarding ApiKeys in general.

**Comment by mh-trimble** [comment]
Here is the link to the "javadocs" in the readme
https://github.com/googleapis/google-cloud-java/blob/aa25967b6482694f8ad41011c023a87bc31686e8/java-maps-places/README.md?plain=1#L191

**Comment by blakeli0** [comment]
I agree it's not straightforward to set up API Keys at this moment, and it is not documented either. I created a separate issue to track the effort of documentation.

**Comment by lqiu96** [comment]
Closing this issue as the maps documentation cannot be published for now. Will update with more information when we get it.
